almalinux-deploy.sh

```diff
@@ -294,7 +294,7 @@ migrate_from_centos() {
                     oracle-backgrounds oracle-logos oracle-indexhtml \
                     oracle-logos-ipa oracle-logos-httpd ; do
         if rpm -q "${pkg_name}" &>/dev/null; then
-            alma_pkg="$(echo $pkg_name | sed 's#centos\|oracle#almalinux#')"
+            alma_pkg="${pkg_name//centos\|oracle/almalinux}"
```

Comment:
Unfortunately, it doesn't work in this way:
```sh
~]$ pkg_name='centos-release'
~]$ echo "${pkg_name//centos\|oracle/almalinux}"
centos-release
~]$ pkg_name='oracle-release'
~]$ echo "${pkg_name//centos\|oracle/almalinux}"
oracle-release
```
and what is expected:
```sh
~]$ pkg_name='centos-release'
~]$ echo "$(echo $pkg_name | sed 's#centos\|oracle#almalinux#')"
almalinux-release
~]$ pkg_name='oracle-release'
~]$ echo "$(echo $pkg_name | sed 's#centos\|oracle#almalinux#')"
almalinux-release
```

I propose disabling the `SC2001` check for this line until we know how to implement it better:
```sh
# shellcheck disable=SC2001
alma_pkg="$(echo $pkg_name | sed 's#centos\|oracle#almalinux#')"
```